help/Teste/UsuarioTestes.cs

```diff
@@ -24,7 +24,7 @@ public void DeveriaSalvarProfessorChamandoORepositoryUmaVez()
             sut.Cadastrar("Robson", "Junior", "robsonjunior@mail.com", "123456789", "123456789");
 
             //assert
-            usuarioRepositoryMock.Verify(x => x.Salvar(It.IsAny<Usuario>()), Times.Once);
+            usuarioRepositoryMock.Verify(x => x.Cadastrar(It.IsAny<Usuario>()), Times.Once);
         }
```

Comment:
Okay, agora só falta colocar as possibilidades de erros nos testes. Ex.: "Deveria não cadastrar caso senha repetida for diferente" .

Reply:
Esquece os teste por hora, eles não estão configurado, estou aguardando explicação do @lucasnobredev